### DISCUSSION TITLE
Update CRM

### DISCUSSION THREAD

**Discussion by gsportelli73** [YetiForceCompany/YetiForceCRM#14443] [General]
Hello guys, i have a question.
My client has yeti 5.3 last update installed on the vps.
Why i don't automatically update to a new version ?
I see that latest stable version is 6.1.
Thank for attenion

**Comment by mariuszkrzaczkowski** [comment]
The system does not have automatic updates, it is too complicated a process, we plan to do automatic installation of the service pack

https://gitdeveloper.yetiforce.com/index.php?parent=Settings&module=Updates&view=Index
